src/sentry/static/sentry/app/views/eventsV2/eventDetails/transaction/opsBreakdown.tsx

```diff
@@ -170,7 +187,7 @@ class EventBreakdown extends React.Component<Props> {
         {results.ops.map(currOp => {
           const {name, percentage, totalDuration} = currOp;
           const durLabel = Math.round(totalDuration * 1000 * 100) / 100;
-          const pctLabel = Math.round(percentage * 100);
+          const pctLabel = isFinite(percentage) ? Math.round(percentage * 100) : '∞';
```

Comment:
@doralchan I took the liberty to use the `∞` unicode for the infinity case (i.e. division by 0 case).